src/AlgebraicGeometry/ToricVarieties/ToricSchemes/attributes.jl

```diff
@@ -49,7 +107,7 @@ with default covering
     1: normal, affine toric variety
     2: normal, affine toric variety
     3: normal, affine toric variety
-  in the coordinate(s)
+  in the coordinate(s)Od
```

Comment:
This doesn't look right.

Reply:
Yes that looks weird. I am surprised it did not lead to errors. I fixed it.